src/main/java/com/onionshop/controllers/ProjectStateController.java

```diff
@@ -168,35 +161,39 @@ protected void onToolSizeSliderMove() {
         //toolSize = toolSizeSlider.getValue();
     }
 
-    /**
-     * Takes in a ToolbarEvent, update the tool accordingly
-     * To be implemented after phase0
-     */
-    public void toolUpdate(){
-
-    }
-
     /**
      * Takes in an event that changes the size of brush, update the brush size accordingly
      *
      */
     public void brushSizeUpdate(){
+
     }
 
+    @FXML
+    protected void onBrushPenClick() {
+        Pen currentToolState = new Pen("round", 10);
+        ToolStateManager.getInstance().setCurrentToolState(currentToolState);
+    }
 
-    public void onEraserClick(ActionEvent actionEvent) {
-        eraser.setText("Eraser: Selected");
-        brushPen.setText("Pen");
+    @FXML
+    public void onEraserClick() {
+        Eraser currentToolState = new Eraser("round", 10);
+        ToolStateManager.getInstance().setCurrentToolState(currentToolState);
     }
 
-    
 
-    public void onLineToolClick(ActionEvent actionEvent) {
```

Comment:
yeah just go back and add comment headers to all of these

Reply:
will keep in mind for tmr's push ⭐️⭐️⭐️